### PR TITLE
Completed html-tables exercise-David Gonzalez

### PR DESCRIPTION
**Pull request by David-Gonzalez11**
Tables

### CODE REVIEW

File: html-tables/index.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    <title>Student Grade Table</title>
+  </head>
+  <body>
+    <h1>Student Grade Table</h1>
+    <table>
+<tr>
+  <th>Student Name</th>
+  <th>Course</th>
+  <th>Grade</th>
+      </tr>
```
Comment (codingcodymiller) [inline html-tables/index.html:13]: This should be in a `thead`

File: html-tables/index.html [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+      <tr>
+        <th>Grace Hopper</th>
```
Comment (codingcodymiller) [inline html-tables/index.html:15]: This should be a `td`

File: html-tables/index.html [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+        <td>Computer Science</td>
+        <td>100</td>
+      </tr>
+
+      <tr>
+        <th>Alan Turning</th>
```
Comment (codingcodymiller) [inline html-tables/index.html:21]: This should be a `td`

File: html-tables/index.html [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+        <td>Mathematics</td>
+        <td>100</td>
+      </tr>
+      <tr>
+        <th>Douglas Crockford </th>
```
Comment (codingcodymiller) [inline html-tables/index.html:26]: This should be a `td`

File: html-tables/index.html [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+        <td>JSON</td>
+        <td>98</td>
+      </tr>
+      <tr>
+        <th>Tim Berners-Lee</th>
```
Comment (codingcodymiller) [inline html-tables/index.html:31]: This should be a `td`

File: html-tables/index.html [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+        <td>HTML</td>
+        <td>9001</td>
+      </tr>
+      <tr>
+        <th>Linus Torvalds</th>
```
Comment (codingcodymiller) [inline html-tables/index.html:36]: This should be a `td`

File: html-tables/index.html [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+        <td>Git</td>
+        <td>90</td>
+      </tr>
```
Comment (codingcodymiller) [inline html-tables/index.html:39]: This should be in a `tbody`

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey David! In addition to my other feedback, please fix your indentation in this file.

**Comment by David-Gonzalez11** [comment]
fixed isasues

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**